auth-web/src/components/auth/common/NumberedCompanyTooltip.vue

```diff
@@ -1,24 +1,40 @@
 <template>
   <v-tooltip top max-width="450px" light content-class="tooltip">
     <template v-slot:activator="{ on }">
-      <span v-on="on" class="tooltip-text">Numbered Benefit Company</span>
+      <span v-on="on" class="tooltip-text" v-if="enableBcCccUlc">Numbered Company</span>
+      <span v-on="on" class="tooltip-text" v-else>Numbered Benefit Company</span>
     </template>
     <v-card class="tooltip-content">
-      <h3 class="mb-3">Numbered Benefit Company</h3>
-      <span>A Benefit Company can choose to use as its name the incorporation number of the company followed by “B.C.
-        Ltd.” The incorporation number is assigned by the Business Registry after the Incorporation Application is filed
-        and the company is incorporated.</span>
+      <div v-if="enableBcCccUlc">
+        <h3 class="mb-3">Numbered Company</h3>
+        <span>A Company can choose to use as its name the incorporation number of the company followed by “B.C. Ltd.”,
+          "B.C. Unlimited Liability Company", or "B.C. Community Contribution Company." The incorporation number is
+          assigned by the Business Registry after the Incorporation Application is filed and the company is
+          incorporated.</span>
+      </div>
+      <div v-else>
+        <h3 class="mb-3">Numbered Benefit Company</h3>
+        <span>A Benefit Company can choose to use as its name the incorporation number of the company followed by “B.C.
+          Ltd.” The incorporation number is assigned by the Business Registry after the Incorporation Application is
+          filed and the company is incorporated.</span>
+      </div>
     </v-card>
   </v-tooltip>
 </template>
 
 <script lang="ts">
 import { Component } from 'vue-property-decorator'
+import { LDFlags } from '@/util/constants'
+import LaunchDarklyService from 'sbc-common-components/src/services/launchdarkly.services'
 import Vue from 'vue'
 
 @Component({})
 export default class NumberedCompanyTooltip extends Vue {
+  private get enableBcCccUlc (): boolean {
```

Comment:
Local getters can be public since code outside this class can call it safely.

Again, low priority (no impact atm I think). No need to change.

Reply:
This comment applies to the other Vue files if you do make an update.

Reply:
Done for all

Reply:
PS You can leave out `public` as it's public by default.